### ISSUE TITLE
Allow custom scan delay option in config

### ISSUE THREAD

**Issue by trippyhat** [favll/pogom#160]
It would be nice if we had a way to specify how long we want the threads to sleep for in the config file, similar to what is seen in other maps (ie `scan delay = 12 seconds`)

**Comment by Pkzz** [comment]
Agreed. With Niantic's never-seem-to-end policy of increasing time bewteen scans, this would be great even to avoid unnecessary updates.
